packages/config/config/devices/0x0234/zdb5100.json

```diff
@@ -1,827 +1,582 @@
-		},
-		"20": {
-			"label": "Pushbutton 1 LED indication",
-			"description": "This parameter specifies how the LED indication is controlled.",
-			"valueSize": 1,
+			"unit": "10 ms",
+			"defaultValue": 50
+		},
+		"14[0xff000000]": {
+			"$import": "templates/logic_group_template.json#global_brightness",
+			"label": "Global brightness Red LED"
+		},
+		"14[0xff0000]": {
+			"$import": "templates/logic_group_template.json#global_brightness",
+			"label": "Global brightness Green LED"
+		},
+		"14[0xff00]": {
+			"$import": "templates/logic_group_template.json#global_brightness",
+			"label": "Global brightness Blue LED"
+		},
+		"14[0xff]": {
+			"$import": "templates/logic_group_template.json#multilevel_disabled_byte"
+		},
+		"15[0x01]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 2 secure message",
+			"valueSize": 2
+		},
+		"15[0x02]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 3 secure message",
+			"valueSize": 2
+		},
+		"15[0x04]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 4 secure message",
+			"valueSize": 2
+		},
+		"15[0x08]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 5 secure message",
+			"valueSize": 2
+		},
+		"15[0x10]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 6 secure message",
+			"valueSize": 2
+		},
+		"15[0x20]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 7 secure message",
+			"valueSize": 2
+		},
+		"15[0x40]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 8 secure message",
+			"valueSize": 2
+		},
+		"15[0x80]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 9 secure message",
+			"valueSize": 2
+		},
+		"15[0x100]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 10 secure message",
+			"valueSize": 2
+		},
+		"15[0x0200]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 11 secure message",
+			"valueSize": 2
+		},
+		"15[0x0400]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 12 secure message",
+			"valueSize": 2
+		},
+		"15[0x0800]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 13 secure message",
+			"valueSize": 2
+		},
+		"15[0x1000]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 14 secure message",
+			"valueSize": 2
+		},
+		"15[0x1fff]": {
+			"label": "Messages in all association groups are sent as secure",
```

Comment:
see my other comment regarding this, "all" just means the other partials referencing the same paramter are set to 1

Reply:
This can be removed from the code. references overlapping bit masks